src/api/db/migrate/20170905081525_add_has_secure_password_support.rb

```diff
@@ -4,9 +4,18 @@ def change
     rename_column :users, :password_hash_type, :deprecated_password_hash_type
     rename_column :users, :password_salt, :deprecated_password_salt
 
-    change_column :users, :deprecated_password, :string, null: true, default: nil, after: :realname
-    change_column :users, :deprecated_password_hash_type, :string, null: true, default: nil, after: :deprecated_password
-    change_column :users, :deprecated_password_salt, :string, null: true, default: nil, after: :deprecated_password_hash_type
+    reversible do |dir|
```

Comment:
Why not using the ``up`` and ``down`` methods we already use everywhere?

Reply:
I think both are valid options. And in this case we have some parts that are reversible. So I prefer to have everything in 'change' and use the reversible block for things we have to explicitly handle for rollbacks.

Reply:
@ChrisBr please review again. Thanks